slash/runner.py

```diff
@@ -120,10 +119,6 @@ def _run_single_test(test, test_iterator):
 
                 except context.session.get_skip_exception_types():
                     pass
-                except INTERRUPTION_EXCEPTIONS:
```

Comment:
@ayalash oops, my bad... Was sure the previous code didn't reraise it... ignore it.